2023-03-23 15:02:56.298 CDT [37878] LOG:  starting PostgreSQL 16devel on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0, 64-bit
2023-03-23 15:02:56.298 CDT [37878] LOG:  listening on IPv6 address "::1", port 5432
2023-03-23 15:02:56.298 CDT [37878] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2023-03-23 15:02:56.300 CDT [37878] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2023-03-23 15:02:56.305 CDT [37881] LOG:  database system was shut down at 2023-03-23 15:02:42 CDT
2023-03-23 15:02:56.311 CDT [37878] LOG:  database system is ready to accept connections
2023-03-23 15:03:16.383 CDT [37993] ERROR:  database "test" already exists
2023-03-23 15:03:16.383 CDT [37993] STATEMENT:  CREATE DATABASE test;
2023-03-23 15:03:21.351 CDT [38017] ERROR:  database "test2" already exists
2023-03-23 15:03:21.351 CDT [38017] STATEMENT:  CREATE DATABASE test2;
2023-03-23 15:03:49.435 CDT [38165] ERROR:  relation "test1" does not exist at character 15
2023-03-23 15:03:49.435 CDT [38165] STATEMENT:  SELECT * FROM test1 WHERE t = $1
2023-03-23 15:04:11.274 CDT [38299] ERROR:  database "test1" already exists
2023-03-23 15:04:11.274 CDT [38299] STATEMENT:  CREATE DATABASE test1;
2023-03-23 15:04:58.095 CDT [37878] LOG:  received fast shutdown request
2023-03-23 15:04:58.096 CDT [37878] LOG:  aborting any active transactions
2023-03-23 15:04:58.097 CDT [37878] LOG:  background worker "logical replication launcher" (PID 37884) exited with exit code 1
2023-03-23 15:04:58.097 CDT [37879] LOG:  shutting down
2023-03-23 15:04:58.098 CDT [37879] LOG:  checkpoint starting: shutdown immediate
2023-03-23 15:04:58.103 CDT [37879] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.002 s, sync=0.001 s, total=0.006 s; sync files=2, longest=0.001 s, average=0.001 s; distance=0 kB, estimate=0 kB; lsn=0/20A9D60, redo lsn=0/20A9D60
2023-03-23 15:04:58.110 CDT [37878] LOG:  database system is shut down
2023-03-23 15:05:03.481 CDT [38693] LOG:  starting PostgreSQL 16devel on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0, 64-bit
2023-03-23 15:05:03.482 CDT [38693] LOG:  listening on IPv6 address "::1", port 5432
2023-03-23 15:05:03.482 CDT [38693] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2023-03-23 15:05:03.483 CDT [38693] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2023-03-23 15:05:03.487 CDT [38696] LOG:  database system was shut down at 2023-03-23 14:45:59 CDT
2023-03-23 15:05:03.492 CDT [38693] LOG:  database system is ready to accept connections
2023-03-23 15:10:03.587 CDT [38694] LOG:  checkpoint starting: time
2023-03-23 15:10:03.995 CDT [38694] LOG:  checkpoint complete: wrote 7 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.404 s, sync=0.002 s, total=0.408 s; sync files=6, longest=0.001 s, average=0.001 s; distance=0 kB, estimate=0 kB; lsn=0/1CAACD0, redo lsn=0/1CAAC98
2023-03-23 15:13:19.776 CDT [38693] LOG:  received fast shutdown request
2023-03-23 15:13:19.776 CDT [38693] LOG:  aborting any active transactions
2023-03-23 15:13:19.778 CDT [38693] LOG:  background worker "logical replication launcher" (PID 38699) exited with exit code 1
2023-03-23 15:13:19.778 CDT [38694] LOG:  shutting down
2023-03-23 15:13:19.779 CDT [38694] LOG:  checkpoint starting: shutdown immediate
2023-03-23 15:13:19.782 CDT [38694] LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.001 s, sync=0.001 s, total=0.004 s; sync files=0, longest=0.000 s, average=0.000 s; distance=0 kB, estimate=0 kB; lsn=0/1CAAD80, redo lsn=0/1CAAD80
2023-03-23 15:13:19.789 CDT [38693] LOG:  database system is shut down
